__tests__/types.d.ts

```diff
@@ -42,3 +42,18 @@ export interface Photo {
   setImg: (img: object) => void;
   setDescription: (description: string) => void;
 }
+export interface Permission {
+  user: User;
+  shelter: Shelter;
+  value: number;
+}
+export interface PermissionsList {
+  permissions: Permission[];
+  removePermission: (index: int) => void;
```

Comment:
Why do we pass an index? How would it be known? I think it's better to pass `Permission` type object.